com.unity.visualeffectgraph/Editor/Core/VFXLibrary.cs

```diff
@@ -489,21 +482,66 @@ private static void LoadSRPBindersIfNeeded()
             }
         }
 
+        private static bool unsupportedSRPWarningIssued = false;
+
+        private static void LogUnsupportedSRP(VFXSRPBinder binder, bool forceLog)
+        {
+            if (binder == null && (forceLog || !unsupportedSRPWarningIssued))
+            {
+                Debug.LogWarning("The Visual Effect Graph is supported in the High Definition Render Pipeline (HDRP) and the Universal Render Pipeline (URP). Please assign your chosen Render Pipeline Asset in the Graphics Settings to use it.");
```

Comment:
it should also mention the Quality Settings (A valid Render Pipeline asset needs to be assigned in the Graphics Settings or the Quality Settings.)

Reply:
So what would be the text? "Please assign your chosen Render Pipeline Asset in the Graphics or Quality Settings to use it" ?

Reply:
This is what message the tech writing team gave to me for my feature "Project graphics settings do not refer to a URP Asset. Check the settings: Graphics > Scriptable Render Pipeline Settings, Quality > Render Pipeline Asset."
maybe it could be tweaked for your case to:
"The Visual Effect Graph is supported in the High Definition Render Pipeline (HDRP) and the Universal Render Pipeline (URP). Check the settings: Graphics > Scriptable Render Pipeline Settings, Quality > Render Pipeline Asset."